views/scss/item-creator.scss

```diff
@@ -794,6 +794,12 @@ img.dragging.ui-draggable-dragging {
                     min-width: 100%;
                 }
 
+                .qti-include {
+                    img {
+                        min-width: initial;
```

Comment:
Checked, and bad luck. [initial is not supported](https://developer.mozilla.org/en-US/docs/Web/CSS/initial#Browser_compatibility) either. Did you check it under IE? On my testing env it is not working.

Reply:
Sorry I don't test again, VM open ralentize all. Bad, not check docs too.

I check with `inherit` and I push changes

![imagen](https://user-images.githubusercontent.com/34692207/92125520-68dd5300-edff-11ea-8270-497fda8d161b.png)